DistributionPackages/Neos.NeosIo/Configuration/NodeTypes.Content.Column.yaml

```diff
@@ -0,0 +1,46 @@
+'Neos.NeosIo:Content.Column':
+  superTypes:
+    Neos.Neos:ContentCollection: true
+    Neos.NeosIo:Mixin.ButtonStyles: true
+  ui:
+    label: Column
+    icon: 'icon-columns'
+    inspector:
+      groups:
+        column:
+          label: Column
+        appearance:
+          label: Appearance
+  properties:
+    moreLink:
+      type: string
+      ui:
+        label: 'More link'
+        reloadIfChanged: true
+        inspector:
+          group: column
+          editor: Neos.Neos/Inspector/Editors/LinkEditor
+          position: 10
+    moreLinkLabel:
+      type: string
+      defaultValue: 'Read more'
+      ui:
+        inlineEditable: true
+        inline:
+          editorOptions:
+            autoparagraph: false
+        inspector:
+          group: column
+          type: string
+  constraints:
+    nodeTypes:
+      '*': FALSE
+      'Neos.NodeTypes:Text': TRUE
```

Comment:
lowercase bool values

Reply:
Covert then all? Because seems that the most are in uppercase